fdbserver/Knobs.cpp

```diff
@@ -541,7 +541,12 @@ ServerKnobs::ServerKnobs(bool randomize, ClientKnobs* clientKnobs, bool isSimula
 	// Fast Restore
 	init( FASTRESTORE_FAILURE_TIMEOUT,                          3600 );
 	init( FASTRESTORE_HEARTBEAT_INTERVAL,                         60 );
-	init( FASTRESTORE_SAMPLING_PERCENT,                            1 ); if( randomize && BUGGIFY ) { FASTRESTORE_SAMPLING_PERCENT = deterministicRandom()->random01() * 100; }
+	init( FASTRESTORE_SAMPLING_PERCENT,                            1 ); if( randomize ) { FASTRESTORE_SAMPLING_PERCENT = deterministicRandom()->random01() * 100; }
```

Comment:
All parallel restore tests do not use BUGGIFY. So I remove it for testing.
In the next PR, I also removed the BUGGIFY for other fast restore knobs.

Reply:
I didn't remove the BUGGIFY for other knobs in this PR because I was not sure if the code works when the other knobs change. 
The following PRs will remove the buggify and pass correctness test.